data_summaries/MathDial.json

```diff
@@ -0,0 +1,57 @@
+{
+    "mathdial": {
+        "Unique Dataset Identifier": "mathdial",
+        "Dataset Name": "mathdial",
+        "Paper Title": "MathDial: A Dialogue Tutoring Dataset with Rich Pedagogical Properties Grounded in Math Reasoning Problems",
+        "Dataset URL": "https://github.com/eth-nlped/mathdial/blob/main/data/train.jsonl",
+        "GitHub URL": "https://github.com/eth-nlped/mathdial",
+        "Hugging Face URL": "",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2305.14536",
+        "Semantic Scholar Corpus ID": 258865403,
+        "Collection": "MathDial",
+        "Collection URL": "https://eth-nlped.github.io/mathdial/",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Logical and Mathematical Reasoning",
+            "Dialogue Generation",
+            "Dialogue Tutoring"
+        ],
+        "Text Sources": [
+            "Math Word Problems"
+        ],
+        "Model Generated": [
+            "OpenAI GPT-3.5-Turbo"
+        ],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "GSM8K"
+        ],
+        "Creators": [
+            "ETH Zürich",
+            "Technische Universitat Darmstadt",
+            "Nanyang Technological University"
+        ],
+        "Licenses": [
+            {
+                "License": "CC BY-SA 4.0",
+                "License URL": "https://github.com/eth-nlped/mathdial#example"
+            },
```

Comment:
Also add the GSM8K MIT license: https://huggingface.co/datasets/gsm8k#licensing-information

Reply:
Great! Also, I will expand the text sources once the other math-related PR gets merged. I will add most of the sources in Orca Math, so I don't want to duplicate the constants here.

Reply:
Should be ready to make this change and merge?